models/UNet.py

```diff
@@ -96,4 +96,19 @@ def forward(self, X):
         decoder3_out = self.decoder3(torch.cat([decoder4_out, encoder4_out], 1))
         decoder2_out = self.decoder2(torch.cat([decoder3_out, encoder2_out], 1))
         decoder1_out = self.decoder1(torch.cat([decoder2_out, encoder1_out], 1))
+        self.activs = [X, encoder1_out, encoder1_out_pooled,
```

Comment:
Not sure if we want to save all of these activations or not? This could end up creating a massive amount of data if we're not careful.